src/MassTransit.Abstractions/IMetricsTagsFeature.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace MassTransit
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Provides access to tags added to metrics.
+    /// </summary>
+    public interface IMetricsTagsFeature
```

Comment:
Feature isn't used anywhere in MassTransit, so using the same name isn't the right name.

`MetricsContext` perhaps is more aligned, and could be accessed via `TryGetPayload<MetricsContext>()` on any send/consume/publish context.

Reply:
It makes sense. I'll make these changes.